spec/integration/systems_spec.rb

```diff
@@ -45,7 +45,7 @@
     end
   end
 
-  path "#{Settings.path_prefix}/#{Settings.app_name}/systems/{id}" do
+  path '/systems/{id}' do
     get 'Retrieve a system' do
       tags 'host'
       description 'Lists all hosts requested'
```

Comment:
Can you change also description and operation ID for this systems/{id}. Seems it's copied from systems endpoint